### PR TITLE
#997 Removed hard-coded references to keptn events

### PR DESCRIPTION
**Pull request by johannes-b**
- Removed hard-coded references to keptn events
- Clean-up of defineCredentials.sh scripts since prompts for GitHub credentials became obsolete

### CODE REVIEW

File: installer/manifests/keptn/core-distributors.yaml [hunk 1]
```diff
@@ -140,7 +139,7 @@ spec:
             - name: PUBSUB_URL
               value: 'nats://keptn-nats-cluster'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.tests.finished'
```
Comment (christian-kreuzberger-dtx) [inline installer/manifests/keptn/core-distributors.yaml]: IMHO the `sh.keptn.event.tests.finished` is correct?

In go-utils we have:
```go
// TestsFinishedEventType is a CloudEvent for indicating that tests have finished
const TestsFinishedEventType = "sh.keptn.event.tests.finished"

// TestFinishedEventType_0_5_0_Compatible is a CloudEvent for indicating that tests have finished
const TestFinishedEventType_0_5_0_Compatible = "sh.keptn.events.tests-finished"
```

File: lighthouse-service/deploy/distributor.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ spec:
             - name: PUBSUB_URL
               value: 'nats://keptn-nats-cluster'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.tests.finished'
```
Comment (christian-kreuzberger-dtx) [inline lighthouse-service/deploy/distributor.yaml]: Same here. `tests.finished` should be the correct one.

File: installer/manifests/keptn/core-distributors.yaml (hunk 1 above)
Comment (johannes-b) [reply installer/manifests/keptn/core-distributors.yaml]: I removed the `TestFinishedEventType_0_5_0_Compatible` as we want no backward support of changed event types. Thus, I changed the `sh.keptn.event.tests.finished` event to  sh.keptn.events.tests-finished` , which follows the naming convention of others. 
```
// TestsFinishedEventType is a CloudEvent for indicating that tests have finished
const TestsFinishedEventType = "sh.keptn.events.tests-finished"
```

File: lighthouse-service/deploy/distributor.yaml (hunk 2 above)
Comment (johannes-b) [reply lighthouse-service/deploy/distributor.yaml]: see above

File: installer/manifests/keptn/core-distributors.yaml (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply installer/manifests/keptn/core-distributors.yaml]: Thanks for clarification.

**Pull request merged**